libraries/chain/include/eosio/chain/block_summary_object.hpp

```diff
@@ -34,4 +34,5 @@ namespace eosio { namespace chain {
 
 CHAINBASE_SET_INDEX_TYPE(eosio::chain::block_summary_object, eosio::chain::block_summary_multi_index)
 
+// @ignore id
```

Comment:
Does the script fail when the remaining "// @ignore id"s are removed?  There are many of these that you added.

Reply:
Took them out.